VideoLocker/res/layout/row_course_list.xml

```diff
@@ -220,16 +175,35 @@
         </LinearLayout>
     </RelativeLayout>
 
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/course_certified_banner"
+        style="@style/bold_white_text"
+        android:layout_width="wrap_content"
```

Comment:
It has the exact same effect, without adding an unnecessary container. True, it doesn't allow setting the padding without reference to the width, but that doesn't seem to be that much of a big deal.